packages/yoroi-extension/app/utils/formatters.js

```diff
@@ -87,10 +87,10 @@ export function truncateStakePool(addr: string): string {
   return truncateFormatter(addr, 18);
 }
 
-export function truncateAddress(addr: string): string {
+export function truncateAddress(addr: string, cutoff: number = 40): string {
```

Comment:
`truncateAddressShort` function is the same

Reply:
Good catch! Switched to use `truncateAddressShort` and rolled back that change